aspnetcore/fundamentals/environments.md

```diff
@@ -89,6 +89,14 @@ These commands take effect only for the current window. When the window is close
 
 ![ASPNET Core Environment Variable](environments/_static/windows_aspnetcore_environment.png) 
 
+**web.config**
+
+See [Setting environment variables in the ASP.NET Core Module configuration reference](xref:hosting/aspnet-core-module#setting-environment-variables) topic.
```

Comment:
IIRC there wasn't guidance for linking to sections of topics. It discombobulated me here, since I normally just link by topic title. I'll try again on the next commit to fix it.